src/components/credits/Contributor.jsx

```diff
@@ -0,0 +1,17 @@
+import ExternalLink from "@/components/shared/ExternalLink";
+
+export default function Contributor(props) {
+    return (
+        <li>
+            <strong>
+                {props.link ? <NameWithLink name={props.name} link={props.link} /> : props.name}
+            </strong>: {props.children}
+        </li>
+    );
+}
+
+function NameWithLink(props) {
```

Comment:
Same as above and move it above Contributor or better, move it to a seperate file (so one JSX = one component)

Reply:
I put it in that file because there's only one insurance of the component being used

Reply:
That's totally fine but I would still split them.